README.md

```diff
@@ -14,48 +14,49 @@ Headless Chat SDK to build your own chat widget against Dynamics 365 Omnichannel
 Please make sure you have a chat widget configured before using this package or you can follow this [link](https://docs.microsoft.com/en-us/dynamics365/customer-service/add-chat-widget)
 
 ## Table of Contents
+
 - [Live Chat Widget vs. Chat SDK](#live-chat-widget-vs-chat-sdk)
 - [Releases](#releases)
 - [Installation](#installation)
 - [Installation on React Native](#installation-on-react-native)
 - [SDK Methods](#sdk-methods)
-    - [Initialization](#initialization)
```

Comment:
Are the methods under the “SDK Methods” section? That was the intention of the indentations